spec/classes/prosody_spec.rb

```diff
@@ -71,6 +71,16 @@
             with_content(%r{^  str = "twentyfour";})
         }
       end
+
+      context 'with community modules' do
+        let(:params) { { community_modules: %w[coolmod othermod] } }
+
```

Comment:
please also add a `it { is_expected.to compile.with_all_deps }`. And I guess that will fail because there is a typo in the .fixtures.yml and the vcsrepo resource cannot be found (or that codepath is still untested).